Modules/IndividualAssessment/classes/AccessControl/class.ilIndividualAssessmentAccessHandler.php

```diff
@@ -238,4 +238,20 @@ protected function cacheCheckAccessToObj($operation)
 
 		return $this->mass_global_permissions_cache[$iass_id][$user_id][$operation];
 	}
+
+	/**
+	 * Check wheter user is admin.
+	 *
+	 * @return bool
+	 */
+	public function isAdmin()
+	{
+		$global_roles = $this->review->assignedGlobalRoles($this->usr->getId());
+		foreach ($global_roles as $global_role) {
+			if(ilObject::_lookupTitle($global_role) == "Administrator") {
```

Comment:
Please do not use magic strings. In general. People might just rename that role and the stuff breaks.

Reply:
Ain't there something like "isSuperAdmin" on ilAccess, ilRBACReview or similar?

Reply:
Nein

Reply:
you could use 
```php
$rbacsystem->checkAccessOfUser($this->usr->getId(), "write", SYSTEM_FOLDER_ID)
```
i think, only admins has write access to this.... ("Administration" / "general settings")

Reply:
Why don't you check if the $global_role matches SYSTEM_ROLE_ID (global constant)?

Or even better: `$this->review->isAssigned($this->usr->getId(), SYSTEM_ROLE_ID);`

Reply:
Please consider that access permissions in Administration are also under RBAC control and that one can create new local and global roles with READ or WRITE permissions to any administration node. "Only admins have write access" doesn't go far enough.